app/src/main/java/de/danoeh/antennapod/ui/screen/drawer/NavListAdapter.java

```diff
@@ -216,7 +216,7 @@ private void bindNavView(@StringRes int title, int position, NavHolder holder) {
                 holder.count.setText(NumberFormat.getInstance().format(sum));
                 holder.count.setVisibility(View.VISIBLE);
             }
-        } else if (tag.equals(CompletedDownloadsFragment.TAG) && UserPreferences.isEnableAutodownload()) {
+        } else if (tag.equals(CompletedDownloadsFragment.TAG)) {
```

Comment:
Hmm, so that icon would be shown for people who don't use auto-download at all. We need to come up with a better way to indicate this. (also, people don't seem to understand this icon anyway, so maybe it needs to be changed completely)

Reply:
Yes. I have not touched the limitation-part at all, so this shows up when that limit is reached. But this would be expected anyway, so i don't think that is a problem besides the general issue with it.